app/src/main/java/com/nexenio/rxandroidbleserverapp/ExampleActivity.java

```diff
@@ -1,184 +1,186 @@
-        errorSnackbar.setAction(R.string.action_grant_permission, v -> requestMissingPermissions());
-        errorSnackbar.show();
-    }
-
-    @SuppressLint("CheckResult")
-    private void requestMissingPermissions() {
-        rxPermissions.request(Manifest.permission.BLUETOOTH, Manifest.permission.BLUETOOTH_ADMIN)
-                .subscribe(permissionsGranted -> {
-                    if (!permissionsGranted) {
-                        showMissingPermissionError();
-                    }
-                });
-    }
-
-    /*
-        Bluetooth
-     */
-
-    private void checkBluetoothEnabled() {
-        BluetoothAdapter bluetoothAdapter = BluetoothAdapter.getDefaultAdapter();
-        if (bluetoothAdapter == null || !bluetoothAdapter.isEnabled()) {
-            showBluetoothDisabledError();
-        }
-    }
-
-    private void showBluetoothDisabledError() {
-        errorSnackbar.dismiss();
-        errorSnackbar = Snackbar.make(constraintLayout, R.string.error_bluetooth_disabled, Snackbar.LENGTH_INDEFINITE);
-        errorSnackbar.setAction(R.string.action_enable, v -> enableBluetooth());
-        errorSnackbar.show();
-    }
-
-    private void enableBluetooth() {
-        Intent enableBtIntent = new Intent(BluetoothAdapter.ACTION_REQUEST_ENABLE);
-        startActivityForResult(enableBtIntent, REQUEST_ENABLE_BLUETOOTH);
-    }
-
-    /*
-        Utilities
-     */
-
-    private void performTroubleshooting(@NonNull Throwable throwable) {
-        checkPermissions();
-        checkBluetoothEnabled();
-    }
-
-    private void showTemporaryMessage(@NonNull Throwable throwable) {
-        if (throwable.getMessage() != null) {
-            showTemporaryMessage(throwable.getMessage());
-        } else {
-            showTemporaryMessage(throwable.getClass().getSimpleName());
-        }
-    }
-
-    private void showTemporaryMessage(@NonNull String message) {
-        Toast.makeText(this, message, Toast.LENGTH_LONG).show();
-    }
-
-}
+package com.nexenio.rxandroidbleserverapp;
+
+import com.google.android.material.snackbar.Snackbar;
+
+import android.Manifest;
+import android.annotation.SuppressLint;
+import android.bluetooth.BluetoothAdapter;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.os.Bundle;
+import android.widget.Toast;
+import android.widget.ToggleButton;
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.constraintlayout.widget.ConstraintLayout;
+import androidx.core.content.ContextCompat;
+import androidx.lifecycle.ViewModelProvider;
+
+import java.util.concurrent.TimeUnit;
+
+import io.reactivex.rxjava3.core.Completable;
+import io.reactivex.rxjava3.core.Flowable;
+
+import timber.log.Timber;
+
+import com.tbruyelle.rxpermissions3.RxPermissions;
+
+
+public class ExampleActivity extends AppCompatActivity {
```

Comment:
Any idea why this is showing as a completely new file?

Reply:
Unfortunately not. I tried to fix this by starting a new branch from your latest commit on the master branch and applying the changes I made once again on that branch, but that did not fix it